README.md

```diff
@@ -158,23 +209,25 @@ Results for applying pruning to reduce the parameters of the following models by
 | ResNet50 [2]    | 75.1                 | 72.4                  |
 | DenseNet121 [3] | 74.44                | 71.71                 |
 
-
-## Trouble Shooting
+## Troubleshooting and Community
 
 If the accuracy degradation of the quantized model is too large for your application, check out the [Quantization Troubleshooting](https://github.com/sony/model_optimization/tree/main/quantization_troubleshooting.md)
```

Comment:
"If the accuracy degradation of the quantized model is too large for your application" -->
"If you encountered large accuracy degradation from the quantized model returned by MCT..."

Reply:
fixed